webui/src/lib/components/repository/tree.jsx

```diff
@@ -261,36 +261,36 @@ export const URINavigator = ({ repo, reference, path, relativeTo = "", pathURLBu
     );
 };
 
-const GetStarted = ({ onUpload }) => {
+const GetStarted = ({ onUpload, onImport }) => {
     return (
         <Container className="m-4 mb-5">
             <h2 className="mt-2">To get started with this repository:</h2>
 
             <Row className="pt-2 ml-2">
                 <DotIcon className="mr-1 mt-1"/>
-                <Button variant="link" onClick={onUpload}>Upload</Button>&nbsp;an object.
+                &nbsp;<Button variant="link" onClick={onImport}>Import</Button>&nbsp;data from the object store. Or, see the&nbsp;<a href="https://docs.lakefs.io/setup/import.html" target="_blank" rel="noopener noreferrer">docs</a>&nbsp;for other ways to import data to your repository.
```

Comment:
How hard would it be to replace `the object store` from the actual object store name (e.g. S3, GCS, Azure) depending on the lakeFS type? I think it would make it much clearer.

Reply:
done